fix(genapis): shorten and merge questions MTA-7253 (#6758)
scaleway/docs-content: pages/generative-apis/faq.mdx

Reviewer: Shouldn't this be "bash" for the snippet to be properly color-coded?

Author: Yes, I'm changing it.

### Is Generative APIs suitable for real-time applications?
Yes, *Dedicated Deployment* is designed for low-latency, high-throughput applications, making it suitable for real-time use cases such as chatbots, recommendation systems, fraud detection, and live video processing.

### Can I fine-tune or retrain my models within Generative APIs?
*Dedicated Deployment* is primarily designed for deploying and running inference workloads. If you need to fine-tune or retrain models, you may need to use a separate training environment, such as [Scaleway’s GPU Instances](/gpu/quickstart/), and then deploy the trained model in Generative APIs - Dedicated Deployment.

## Getting started

### How do I get started with Generative APIs?
To get started with *Serverless*, explore the [Generative APIs Playground](/generative-apis/quickstart/#interacting-with-generative-apis-via-the-playground) in the Scaleway console. For application integration, refer to our [Quickstart guide](/generative-apis/quickstart/), which provides step-by-step instructions on accessing, configuring, and using a Generative APIs endpoint.

> To deploy a model using *Dedicated Deployment*, you can use Scaleway's [console](https://console.scaleway.com/generative-api/deployments) or [API](https://www.scaleway.com/en/developers/api/inference/). You can choose a model from Scaleway’s selection or import your own directly from Hugging Face's repositories, configure [Instance types](/gpu/reference-content/choosing-gpu-instance-type/), set up networking options, and start inference with minimal setup. For details, see the [document about how to create a deployment](/generative-apis/how-to/create-deployment/).

Reviewer: ```suggestion
To deploy a model using *Dedicated Deployment*, you can use the Scaleway [console](https://console.scaleway.com/generative-api/deployments) or [API](https://www.scaleway.com/en/developers/api/inference/). You can choose a model from Scaleway selection or import your own directly from Hugging Face's repositories, configure [Instance types](/gpu/reference-content/choosing-gpu-instance-type/), set up networking options, and start inference with minimal setup. For details, see [how to create a deployment](/generative-apis/how-to/create-deployment/).
```

*Dedicated Deployment* supports both open-source models and your own uploaded proprietary models.

### Which new models will be added to Generative APIs?
We cannot disclose information on planned new models before they are released.
However, you can follow our [Changelog](https://www.scaleway.com/en/docs/changelog/?products=generative-apis) and subscribe to its RSS feed to be notified as soon as new models are available.

If you require a specific model, upvote an existing [Feature Request](https://feature-request.scaleway.com/) or create a new one.

We follow closely your feedback and the latest model releases. For *Serverless* usage, our goal is to provide a curated but balanced model catalog across different modalities (Text, Code, Vision, Audio, etc.), model sizes, and providers.

### What is the model lifecycle for Serverless?
Scaleway is dedicated to updating and offering the latest versions of generative AI models, while ensuring older models remain accessible for a significant time, and also ensuring the reliability of your production applications. Learn more in our [model lifecycle policy](/generative-apis/reference-content/model-lifecycle/).

> ### What are each model's characteristics?

Reviewer: ```suggestion
### What are the characteristics of each model?
```

Reviewer: But the title would probably no longer fit on one line, so feel free to find a more compact solution ✨ (also, changing this might break any link leading to this anchor)

Author: The PM said that he didn't mind breaking anchors, the goal is to shorten. But after reflecting, I will keep the current version as-is for brevity.

Model details can be found in our [Supported models](/generative-apis/reference-content/supported-models) catalog. When the model is open-weight, a link to the public model hub (such as Hugging Face) provides additional details from the model provider. 

For *Serverless* models, not all model and inference characteristics are provided (such as [quantization](https://huggingface.co/docs/optimum/en/concept_guides/quantization), [chat templates](https://huggingface.co/docs/transformers/en/chat_templating), [guided decoding backend](https://huggingface.co/blog/nmmursit/guided-decoding), etc.). However, updates to these characteristics are consistent with our [Model Lifecycle Policy](/generative-apis/reference-content/model-lifecycle/):
- For **Preview** models, characteristics may change, in order to benefit from the latest optimizations (tokens throughput, latency, availability, etc.) while maintaining good accuracy for most use cases.

> - For **Active** models, characteristics by default do not change. If a change occurs that may significantly affect output accuracy, this change will be treated as a potentially breaking change. Users will be notified accordingly so they can take the required actions based on their use case's sensitivity to output. 

Reviewer: ```suggestion
- For **Active** models, characteristics by default do not change. If a change occurs that may significantly affect output accuracy, this change will be treated as a potentially breaking change. Users will be notified accordingly so they can take the required actions based on the sensitivity to output of their use case. 
```

Author: I am changing the text but in a slightly different way. The suggested alternative feels less fluid than the original and may be less ideal from a localization perspective too.

...based on how sensitive their use case is to output

For *Dedicated Deployment*, model versions are fixed and provided in the Deployment parameters.

### Where are the inference servers located?
All models are currently hosted in a secure data center located in Paris, France, operated by [OPCORE](https://www.opcore.com/). This ensures low latency for European users and compliance with European data privacy regulations.

### What Instance types are available for inference?

> *Dedicated Deployment* offers different Instance types optimized for various workloads from Scaleway's [GPU Instances](/gpu/reference-content/choosing-gpu-instance-type/) range.

Reviewer: ```suggestion
*Dedicated Deployment* offers different Instance types optimized for various workloads from Scaleway [GPU Instances](/gpu/reference-content/choosing-gpu-instance-type/) range.
```

> You can select the Instance type based on your model’s computational needs and compatibility.

Reviewer: ```suggestion
You can select the Instance type based on your model computational needs and compatibility.
```

Author: I will rewrite it like this:
... based on the computational needs and compatibility of your model

* **Policy-based**: A condition only restricts the specific IAM policy it is attached to. For network restrictions to be fully effective, ensure each policy providing Generative APIs access rights also includes an IAM condition restricting network access.
* **Owner exception**: Users with [Owner status](/iam/concepts/#owner) possess full administrative rights through global organization policies. Even if your custom policies include IP restrictions, an API key belonging to an Owner can still call Generative APIs from any IP address.

> *Dedicated Deployments* - A Dedicated Deployment can be configured to run in a specific Private Network within a VPC. You can additionally remove any public endpoint's access to ensure your Deployment is not accessible from the internet.

Reviewer: ```suggestion
*Dedicated Deployments* - A Dedicated Deployment can be configured to run in a specific Private Network within a VPC. You can additionally remove any public endpoint access to ensure your Deployment is not accessible from the internet.
```

Author: I will rewrite it like this:
... remove access to any public endpoint to ensure ...
